test/junit/xml.go

```diff
@@ -26,4 +26,13 @@ type TestCase struct {
 	ClassName string   `xml:"classname,attr"`
 	Time      float64  `xml:"time,attr"`
 	Failure   string   `xml:"failure,omitempty"`
+	Error     *Error   `xml:"error,omitempty"`
```

Comment:
NOTE: would not it better to have separate models for our junit representation and for the client representations?